### PR TITLE
Debug Container Data Grid

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -414,5 +414,10 @@ In order to create an image using the Red Hat Data Grid server, it's necessary t
 cekit build --overrides-file dg-override.yaml docker
 ```
 
+### Debug Container Data Grid
```
Comment (ryanemerson) [inline README.md]: @diegolovison Thanks for the PR. Can you add a sentence explaining what this allows the user todo.

**Comment by ryanemerson** [comment]
@diegolovison I'm a bit confused by your command. `DEBUG_PORT` doesn't do anything AFAIK. What are you trying to achieve with this command?

**Comment by diegolovison** [comment]
When running in Infinispan in your local environment the port is: `DEBUG_PORT="${DEBUG_PORT:-8787}"`
When you would like to debug what was deployed in the container you need `DEBUG_PORT="*:8787"`

There is no source code in `infinispan-images` related with `DEBUG_PORT` because `infinispan-images` is calling `server.sh` ( https://github.com/infinispan/infinispan-images/blob/master/modules/org/infinispan/runtime/added/bin/launch.sh#L85 )

`server.sh` is in `infinispan` project. see https://github.com/infinispan/infinispan/blob/master/server/runtime/src/main/server/bin/server.sh

`DEBUG_PORT` is here https://github.com/infinispan/infinispan/blob/master/server/runtime/src/main/server/bin/common.sh#L9

**Comment by ryanemerson** [comment]
@diegolovison Of course. Thanks for the explanation. Could you add a sentence to your commit explaining this as the user is unlikely to be aware of this variable's purpose.

**Comment by diegolovison** [comment]
@ryanemerson Done

**Comment by ryanemerson** [comment]
Thanks @diegolovison

**Pull request merged**